delayed-state-rendering.md

```diff
@@ -0,0 +1,273 @@
+- Feature Name: Delayed State Rendering
+- Start Date: 2019-06-21
+- SEP Status: Draft
+- SEP PR: (leave this empty)
+- Salt Issue: (leave this empty)
+- Author: Joseph Nix, nixjdm at terminallabs dot com, [nixjdm@GitHub](https://github.com/nixjdm)
+
+# Summary
+
+This SEP proposes a way of delaying rendering of portions of states until after other states have executed.
```

Comment:
I agree. The suggested change is not quite accurate though, since the sections render more than just states - and really don't even have to have any. I'll work on it.

Reply:
Perhaps `states` is close enough. It's a lot easier to read. Changed locally.